lib/core/src/Cardano/Wallet/DB/Store/Wallets/Model.hs

```diff
@@ -0,0 +1,87 @@
+{-# LANGUAGE NoMonomorphismRestriction #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+
+{- |
+ Copyright: © 2018-2022 IOHK
+ License: Apache-2.0
+
+Low level meta transactions pure model. Meta transactions are encoded "as" expressed in DB
+tables.
```

Comment:
```suggestion
Pure model for the transactions ('Tx') and metadata about them ('TxMeta')
in a collection of wallets.
```

(🤓  At this level of abstraction, we no longer need to mention the encoding of `TxMeta`.)